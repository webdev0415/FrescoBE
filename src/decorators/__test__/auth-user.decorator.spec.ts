import { AuthUser } from '../auth-user.decorator';

describe('AuthUser', () => {
    it('AuthUser snapshot ', () => {
        expect(AuthUser).toMatchSnapshot();
    });
});
