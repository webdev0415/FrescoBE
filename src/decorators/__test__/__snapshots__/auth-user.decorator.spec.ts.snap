// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthUser AuthUser snapshot  1`] = `[Function]`;
